mitmproxy/addons/__init__.py

```diff
@@ -39,6 +40,7 @@ def default_addons():
         script.ScriptLoader(),
         serverplayback.ServerPlayback(),
         setheaders.SetHeaders(),
+        static_viewer.StaticViewer(),
```

Comment:
Adding `StaticViewer` here as default addons makes the building process fail, because we are importing `mitmproxy.tools.web.app.flow_to_json` as dependency. Maybe we should load it only in `WebMaster`?